content/docs/welcome-guide/setup/setup-from-github.md

```diff
@@ -232,4 +306,15 @@ Registering the O3DE engine enables O3DE projects to find the engine, even when
 
     The O3DE manifest file is `<USER_DIRECTORY>/.o3de/o3de_manifest.json`. The paths to all the registered engines, projects, and more are recorded in this file.
 
+{{% /tab %}}
+{{< /tabs >}}
+
 You are now ready to create a project! For an introduction to project configuration, refer to [Project Creation with Open 3D Engine](/docs/welcome-guide/create).
+
+## Side-by-side engines
+ 
+ In the preceding sections, you learned how to set up and configure the O3DE engine as either a source engine or a pre-built SDK engine. You also have the option of registering both engines in a side-by-side configuration. This is one way to isolate your engine source code development from your project development.
+
+ For example, if the engine source is in `C:\o3de` and the pre-built SDK engine is in `C:\o3de-install`, you can give each engine its own engine name so that you can register both in the O3DE manifest using the `o3de` script's `register` command. To test your engine modifications, you can build a test project using the source engine project creation instructions, which builds the engine in the project directory. When you're ready to update the SDK engine used by your production project, you can build a new version of the SDK engine from `C:\o3de` using the `INSTALL` target. This will update the binaries in `C:\o3de-install`.
+
+To update the name of an engine, open `engine.json` in the engine root directory and change the `engine_name` field. Then run `scripts\o3de.bat register --this-engine` again from that directory. New projects created from this engine will be configured to use the new engine name. To update the configured engine for an existing project, open `project.json` in the project root directory and change the `engine` field to use the new engine name.
```

Comment:
```suggestion
To update the name of an engine, open `engine.json` in the engine root directory and change the `engine_name` field. Then run `scripts\o3de.bat register --this-engine` again from that directory. New projects that you create from this engine will use the new engine name. To update the configured engine for an existing project, open `project.json` in the project root directory and change the `engine` field to use the new engine name.
```
Not sure if "will be configured to use" is a key detail in the third sentence. Add it back if that's important.

Reply:
I think that key detail is still implied by your active-voice rewrite.